packages/instantsearch.js/src/widgets/hits/hits.tsx

```diff
@@ -40,7 +41,7 @@ const withUsage = createDocumentationMessageGenerator({ name: 'hits' });
 const Hits = createHitsComponent({ createElement: h, Fragment });
 
 const renderer =
-  ({
+  <THit extends BaseHit = BaseHit>({
```

Comment:
actually all places that have `extends BaseHits = BaseHit` probably should just be `= Record<string, unknown>` instead